src/memory.js

```diff
@@ -0,0 +1,42 @@
+
+import loadROM from './utils/loadROM.js';
+
+class Memory {
+
+  constructor(rom) {
+
+    if (!rom) {
+      throw new Error('requires ROM');
+    }
+
+    this.data = loadROM(rom);
+  }
+
+  readROM(value) {
+    let romValue;
+    if (typeof value != 'number') {
+      romValue = this.data;
+    }
+    else {
+      romValue = this.data[value];
+    }
+    return romValue;
+  }
+
+  readROMRange(range) {
+    // range to read from
+    if (!range) {
+      throw new Error('please provide a range to return');
+    }
+    const low = arguments[0];
+    const high = arguments[1];
```

Comment:
Could you help me understand the reason the parameters are `range` instead of `low, high`?

Reply:
cause I started with range and didn't go back to clean it up...